atmel-samd/mpconfigport.h

```diff
@@ -107,8 +103,8 @@
 
 // Track stack usage on a debug build. Expose results via ustack module.
 #ifdef MICROPY_DEBUG_MODULES
-#define MICROPY_MAX_STACK_USAGE       (1)
 #endif
+#define MICROPY_MAX_STACK_USAGE       (1)
```

Comment:
Did you want to turn on maxstack use checking permanently, or just for now while debugging? I see you moved it out of the `#ifdef MICROPY_DEBUG_MODULES` .

Reply:
I did it while debugging the HardFault. I've moved it back.